vunit/sim_if/modelsim.py

```diff
@@ -218,13 +218,17 @@ def create_library(self, library_name, path, mapped_libraries=None):
         cfg.set("Library", library_name, path)
         write_modelsimini(cfg, self._sim_cfg_file_name)
 
-    def _get_mapped_libraries(self):
+    def _get_mapped_libraries(self, cfg_file_name=None):
```

Comment:
Is the addition of this argument related to the description of the PR? It seems to be a different feature.

Reply:
This allows the same method to be used for subordinate INI files, see recursive call below on line 229.

Reply:
Ok, I overlooked that the call was recursive.